package.json

```diff
@@ -6,15 +6,13 @@
   "type": "module",
   "homepage": "https://paritytech.github.io/contracts-ui",
   "scripts": {
-    "start": "yarn build && webpack serve --config webpack.dev.cjs --open",
-    "build": "NODE_ENV=development webpack --progress --config webpack.dev.cjs",
-    "build:prod": "NODE_ENV=production webpack --config webpack.prod.cjs",
+    "start": "vite",
+    "build": "tsc && vite build",
     "lint": "yarn eslint --fix .",
     "format": "yarn prettier --write .",
     "clean": "rm -rf dist",
     "deep-clean": "yarn clean && rm -rf node_modules",
-    "regenerate": "yarn deep-clean && yarn && yarn start",
-    "prepare": "husky install"
```

Comment:
looks like we need `postinstall` here https://typicode.github.io/husky/#/?id=yarn-2
assuming the instructions are for yarn 2+